docs/UserGuide.md

```diff
@@ -234,9 +239,14 @@ Refer to [Add a module](#add-module) for the tags that you can use.
 * _To include tags:_ `tag <code> include <tag1> (<tag2>...)`
 * _To remove tags:_ `tag <code> remove <tag1> (<tag2>...)`
 
+<div markdown="span" class="alert alert-warning">
+
+:exclamation: **Caution:** Remember to include /t before each tag so that it can be detected. Refer to our examples.
```

Comment:
Perhaps use a different word for "include" to avoid confusion with the actual command format which contains "include" as well. Maybe "add" would be better?